src/main/sensors/gyro.c

```diff
@@ -145,7 +148,7 @@ PG_RESET_TEMPLATE(gyroConfig_t, gyroConfig,
     .gyro_soft_notch_cutoff_1 = 300,
     .gyro_soft_notch_hz_2 = 200,
     .gyro_soft_notch_cutoff_2 = 100,
-    .checkOverflow = true
+    .checkOverflow = GYRO_OVERFLOW_CHECK_ALL_AXES
```

Comment:
@martinbudden: What is the reason that we are now enabling this even for gyros that don't overflow (i.e. MPU6000)?

Reply:
@mikeller See here https://github.com/betaflight/betaflight/pull/4754#issuecomment-353728018

Reply:
Good point. As a safety feature, having it on by default is the prudent way to go anyway - users should have to disable safety features consciously, and assume the risk for doing so.